src/Shape/Whisker.js

```diff
@@ -0,0 +1,90 @@
+
+import {select} from "d3-selection";
+
+import {assign, BaseClass, constant, elem} from "d3plus-common";
+
+import Path from "./Path";
+
+/**
+    @class Whisker
+    @extends BaseClass
+    @desc Creates SVG lines based on an array of data.
+*/
+export default class Whisker extends BaseClass {
+
+  /**
+      @memberof Whisker
+      @desc Invoked when creating a new class instance, and overrides any default parameters inherited from BaseClass.
+      @private
+  */
+  constructor() {
+
+    super();
+
+    this._name = "Whisker";
+    this._pathConfig = {
+      stroke: constant("black"),
+      strokeWidth: constant(1)
+    };
+
+  }
+
+  /**
+      @memberof Whisker
+      @desc Draws the whisker.
+      @chainable
+  */
+  render() {
+
+    if (this._select === void 0) {
+      this.select(select("body").append("svg")
+        .style("width", `${window.innerWidth}px`)
+        .style("height", `${window.innerHeight}px`)
+        .style("display", "block").node());
+    }
+
+    new Path()
```

Comment:
use `Path` to draw the lines from endpoint to origin, and then nest `this._data` based on `this._endpoint` so that each whisker can have different endpoint shapes